### ISSUE TITLE
[Bug]: PHP won't install while connected via hotspot

### ISSUE THREAD

**Issue by ZacHiler** [beyondcode/herd-community#724]
### Platform

Windows

### Operating system version

Windows 11

### System architecture

Intel (x86)

### Herd Version

1.1.0

### Bug description

I hope this is definitely edge case, but wanted to add something here, especially if someone else came looking with a similar problem.
 
TLDR - PHP does not load properly when installing Herd while using hotspot via mobile (AT&T). You must use WiFi or Ethernet.

I had recently reset my laptop to factory and was traveling, using my hotspot to get things set back up. Herd appeared to install without errors, but upon checking the Herd dashboard. PHP was not running.

I tried to stop all services and restart, still nothing. I also tried to delete everything in the /.config/herd/ folder and run the setup again, but PHP still would not load. I also tried to install another version of PHP, but when accessing the PHP version page, it would just sit there. I tried everything multiple times, both with and without activating the license at the first step, and I also ran the setup with and without 'Run as Admin'. Herd's setup states that you will need to approve actions before it downloads PHP, but that never occurs after the initial install.

### Steps to reproduce

Attempt to install Herd while using mobile hotspot (AT&T).
PHP will not be running upon opening Herd.

**Comment by ZacHiler** [comment]
Also just realized that nvm didn't install correctly either, so I had to fix that as well.

**Comment by sschlein** [comment]
Do you have a program that monitors your bandwith when being on a hotspot and this blocks Herd from downloading these files?

I can't come up with a different reason why this would happen.

**Comment by ZacHiler** [comment]
Not to my knowledge. I did mention that my hotspot is through AT&T, so I'm going to see if the same issue exists with other carriers, T-Mobile and Verizon. That's the only others I have access to through family and friends.

During my initial issue, I did perform a speed test  through fast.com and nothing was out of the ordinary. I had full signal as well. I'm going to do some more testing with this just to see what I can find, but I wanted to make sure anyone else trying this was aware.
